### PR TITLE
Rpi uart

### PR DESCRIPTION
**Pull request by JoshMarangoni**

### CODE REVIEW

File: raspberrypi/uart/uart.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import serial
+import time
+
+ser = serial.Serial("/dev/ttyS0", 115200)
```
Comment (jamesrosstwo) [inline raspberrypi/uart/uart.py:4]: I think some documentation would be good here. What is "/dev/ttyS0"?

Additionally, is the baud rate going to be used anywhere else? might be good to have the baud rate in a settings file

File: raspberrypi/uart/uart.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+def listen():
+	received_data = ser.read()
+	time.sleep(0.03)
```
Comment (jamesrosstwo) [inline raspberrypi/uart/uart.py:8]: Documentation would be good to explain the choice of `0.03` specifically

**Review by jamesrosstwo — CHANGES_REQUESTED** [review]

File: raspberrypi/uart/uart.py (hunk 2 above)
Comment (pedramamani) [reply raspberrypi/uart/uart.py:8]: And place it in settings file

**Review by cmcahoon01 — APPROVED** [review]

File: raspberrypi/uart/uart.py (hunk 1 above)
Comment (JoshMarangoni) [reply raspberrypi/uart/uart.py:4]: That is the "file location" of the UART on Linux. Everything on Linux is a file

Comment (jamesrosstwo) [reply raspberrypi/uart/uart.py:4]: Yes, but the idea is to have that as a comment or something for those who don't know that

**Review by jamesrosstwo — APPROVED** [review]

**Pull request merged**